src/ethereum/mod.rs

```diff
@@ -45,252 +67,274 @@ pub struct Options {
     )]
     pub eip1559: bool,
 
-    #[structopt(
-        short,
-        parse(try_from_str),
-        default_value = "false",
-        env = "SIGNUP_SEQUENCER_MOCK"
-    )]
-    pub mock: bool,
+    /// Maximum number of blocks to pull events from in one request.
+    #[structopt(long, env, default_value = "1000")]
```

Comment:
let's replace `BLOCK_QUERY_RANGE` by `MAX_LOG_BLOCKS` in `.github/workflows/build-test-deploy.yml`